README.md

```diff
@@ -119,72 +151,95 @@ calculations. Before the TLS connection is complete, a shared secret key
 must be negotiated between the server and the client. This key is used
 to encrypt all future communications during the connection.
 
-### **MQTT Connection** 
+### MQTT Connection
 
 After successfully connecting on the TLS level, the board starts
 establishing the MQTT connection. Since the TLS handles authentication
 and security, MQTT does not have to provide a username or password.
 
 <img src=".//media/image8.ppm"/>
 
-Checklist
-=========
+## Prerequisites
+
+- [Setup your Azure IoT hub](https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md)
+- [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases)
+- For the Plug and Play sample, download the Plug and Play DTMI [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)
+- [MPLAB X IDE V5.30 or later](https://www.microchip.com/mplab/mplab-x-ide)
+- [XC16 Compiler v1.50 or later](https://www.microchip.com/mplab/compilers)
+
+## Sample Descriptions
+
+### AzureIotDps.X
+
+Connect the device through Azure IoT Hub Device Provisioning to then connect to your IoT Hub.
```

Comment:
Ok. let's rephrase this like so - "This is the primary sample of this guidance which shows case how to connect the PIC-IoT board to Azure IoT Hub through Device Provisioning Service. As the result, you can view the board telemetry on Azure IoT Explorer."